packages/logs/src/domain/networkError/networkErrorCollection.ts

```diff
@@ -37,17 +37,24 @@ export function startNetworkErrorCollection(configuration: LogsConfiguration, li
   function handleResponse(type: RequestType, request: XhrCompleteContext | FetchResolveContext) {
     if (!configuration.isIntakeUrl(request.url) && (isRejected(request) || isServerError(request.status))) {
       if ('xhr' in request) {
-        computeXhrResponseData(request.xhr, configuration, onResponseDataAvailable)
+        computeXhrResponseData(request.xhr, configuration, (responseData) =>
+          onResponseDataAvailable(responseData, request.handlingStack)
+        )
       } else if (request.response) {
-        computeFetchResponseText(request.response, configuration, onResponseDataAvailable)
+        computeFetchResponseText(request.response, configuration, (responseData) =>
+          onResponseDataAvailable(responseData, request.handlingStack)
+        )
       } else if (request.error) {
-        computeFetchErrorText(request.error, configuration, onResponseDataAvailable)
+        computeFetchErrorText(request.error, configuration, (responseData) =>
+          onResponseDataAvailable(responseData, request.handlingStack)
+        )
       }
     }
 
-    function onResponseDataAvailable(responseData: unknown) {
+    function onResponseDataAvailable(responseData: unknown, handlingStack?: string) {
       const domainContext: LogsEventDomainContext<typeof ErrorSource.NETWORK> = {
         isAborted: request.isAborted,
+        handlingStack,
       }
```

Comment:
**💬 suggestion:** ‏You already have access to `request` in that function, no need to pass `handlingStack` as an extra argument.

Reply:
🤦 of course! thanks